code/__DEFINES/maths.dm

```diff
@@ -117,7 +117,8 @@
 #define TORADIANS(degrees) ((degrees) * 0.0174532925)
 
 /// Gets shift x that would be required the bitflag (1<<x)
-#define TOBITSHIFT(bit) ( log(2, bit) )
+/// We need the round because log has floating-point inaccuracy, and if we undershoot at all on list indexing we'll get the wrong index.
```

Comment:
you should make this a multiline autodoc comment using /** */

Reply:
Pushing back slightly: This runs counter to the style of everything else in this file, and also the example given here: https://github.com/tgstation/tgstation/blob/d1606bcae45dbb102f73bf44e8a4effeb086ce65/.github/guides/AUTODOC.md#documenting-a-variabledefine

Reply:
as long as you can find a way to combine both of those `///`s onto one line or use a multi line comment, it’d be good for me just so you can get both comments in the autodoc

Reply:
FWIW I never use `/** */` and always prefer multiple `///` just because it's ugly

But this change isn't necessary since we do not enforce a max line length